.github/workflows/automerge.yaml

```diff
@@ -1,123 +0,0 @@
-name: automerge auto-cherry-picked pr's
```

Comment:
why we're removing automerge from airgun?

Reply:
it was renamed, missed adding it, added it now.

Reply:
@omkarkhatavkar aren't you missing the commit push? unresolving thread

Reply:
yeah some how I pushed to upstream